src/main/java/uk/gov/pay/adminusers/service/UserServices.java

```diff
@@ -278,11 +282,31 @@ public List<UserEntity> getAdminUsersForService(Integer serviceId) {
         }).collect(Collectors.toList());
     }
 
-    private void changeUserFeatures(UserEntity userEntity, String features) {
-        userEntity.setFeatures(features);
-        userEntity.setUpdatedAt(ZonedDateTime.now(ZoneId.of("UTC")));
+    private void changeUserFeatures(UserEntity userEntity, String features, String op) {
+        if (op.equals("add")) {
+            Set<String> featureSet = createFeatureSet(userEntity);
+            featureSet.addAll(Arrays.asList(features.split(",")));
+            featureSet.remove("");
```

Comment:
Why removing an empty string here?

Reply:
Splitting an empty string gives a list containing an empty string, which when joined up adds an extra comma to the csv. This was just a workaround to prevent that, there may be a better way to do it, or it just needs a comment explaining why it's there